addons/website_theme_install/models/website.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
```

Comment:
this module has been merged into website in https://github.com/odoo/odoo/commit/88e910e187346c095a6f0eeb8be647da1e391035